packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -71,6 +73,10 @@ function toTree(vnode) {
       return toTree(node.child);
     case HostPortal: // 4
       return toTree(node.child);
+    case ContextProvider:
+      return toTree(node.child);
+    case ContextConsumer:
+      return toTree(node.child);
```

Comment:
Looks like the test renderer treats them as fragments https://github.com/facebook/react/pull/12237/  Should we do the same? It's really confusing how both files use the same APIs (`toTree`, `childrenToTree`) but don't use them consistently.